squad/frontend/templates/squad/test_run.html

```diff
@@ -63,64 +63,84 @@
-                <div class="progress-bar progress-bar-success" style="width: {{status.pass_percentage}}%" title="Pass: {{status.pass_percentage|stringformat:"d"}}%">
-                    <span class="sr-only">Pass: {{status.pass_percentage}}% </span>
+        <a name="tests-{{status.suite}}"></a>
+        <h3>{{status.suite}}</h3>
+        {% with tests=status.tests_by_result %}
+            {% if tests.fail %}
+                {% for test in status.tests_by_result.fail %}
+                    <div class='row fail'>
+                        <div class='col-md-3 col-sm-3'>
+                            <i class='fa fa-times'></i>
+                            {{test.name}}
+                        </div>
+                        <div class='col-md-9 col-sm-9'>
+                            {% if test.log %}
+                                <pre>{{test.log}}</pre>
+                            {% else %}
+                                <em>No log available for this test</em>
+                            {% endif %}
+                        </div>
+                    </div>
+                {% endfor %}
+            {% endif %}
+            {% if tests.skip %}
+                <div class='row'>
+                    <div class='col-md-12 col-sm-12'>
+                        <span class="badge alert-warning" data-placement="top" title="Failures">{{tests.skip|length}} skipped</span>
+                        <a class='tests-{{status.id}}-skip' href='#' onclick='$(".tests-{{status.id}}-skip").toggle(); return false'>Show</a>
+                        <a class='tests-{{status.id}}-skip' href='#' onclick='$(".tests-{{status.id}}-skip").toggle(); return false' style='display:none'>Hide</a>
+                    </div>
                 </div>
-                <div class="progress-bar progress-bar-warning" style="width: {{status.skip_percentage}}%" title="Skip: {{status.skip_percentage|stringformat:"d"}}%">
-                    <span class="sr-only">Skip: {{status.skip_percentage}}%</span>
+                <div class='row tests-details tests-details-skip tests-{{status.id}}-skip' style='display: none'>
+                    {% for test in status.tests_by_result.skip %}
+                        <div class='col-md-12 col-sm-12'>
+                            <i class='fa fa-question-circle-o'></i>
+                            {{test.name}}
+                        </div>
+                    {% endfor %}
                 </div>
-                <div class="progress-bar progress-bar-danger" style="width: {{status.fail_percentage}}%" title="Fail: {{status.fail_percentage|stringformat:"d"}}%">
-                    <span class="sr-only">Fail: {{status.fail_percentage}}%</span>
+            {% endif %}
+            {% if tests.pass %}
+                <div class='row'>
+                    <div class='col-md-12 col-sm-12'>
+                        <span class="badge alert-success" data-placement="top" title="Failures">{{tests.pass|length}} passed</span>
```

Comment:
title is incorrect

Reply:
fixed (remove title since it doesn't add anything)